src/main/resources/plugin.yml

```diff
@@ -1,7 +1,7 @@
 name: ${project.name}
 version: ${project.version}
-main: com.popupmc.areaSpawner.AreaSpawner
-api-version: "1.13"
+main: com.popupmc.areaspawner.AreaSpawner
+api-version: 1.15
```

Comment:
api-version should be a string, and I don't think changing it from 1.13 to 1.15 really changes anything in our case but I will leave it like that anyways

Reply:
Most things seem to be parsed as strings in yml if quotes are not added which makes many things appear cleaner in the yml file. This was taken from the official Spigot plugin development page [plugin-yml](https://www.spigotmc.org/wiki/plugin-yml/)

There are various minor api changes between each version and it says if we use anything before 1.15, compatibility needs to be applied. I understand we might not be using anything that requires compatibility but I like to be on the safe side as it's meant to be the current-api version we're working with while the pom.xml is supposed to be the target-api version or the highest version we're working with.

![image](https://user-images.githubusercontent.com/1305564/101045561-61d4f680-354e-11eb-9fa3-2cfb08bbd6c1.png)